docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -1281,6 +1281,129 @@ The configuration is simple, you have a role which is the 'Admin role' the one a
 
 For instance; techs are in the AD group tech support and get the role 'tech support' while registering, let's put 'tech support' as the 'Admin role'. They are allowed to register new devices with the roles 'default', 'voice' and 'guest'. Every time someone with the role 'tech support' will try to register a device on a connection profile where this portal module is active, then the crew memeber will be asked to choose which role to assign to this device.
 
+Portal surveys
+~~~~~~~~~~~~~~
+
+PacketFence has the ability to perform surveys via the captive portal and store the results in dedicated tables in the database.
+
+Setup
+^^^^^
+
+In order for the survey tables to be created automatically based on the definition of your surveys, you must grant create and alter rights to the database user defined in pf.conf. By default this user is 'pf'. On your database, connect to the MySQL CLI as root and execute the following:
+
+  mysql> GRANT CREATE,ALTER ON pf.* TO 'pf'@'%';
+  mysql> GRANT CREATE,ALTER ON pf.* TO 'pf'@'localhost';
```

Comment:
I would prefer not to add those privileges to the main pf user as a security issues.
Here are a list of solutions of possible alternative approaches.

* Create a dedicated user that is allowed to create and alter tables.
   That user will only be used when configuring the database.

* Have a dedicated database for the surveys that the pf user has permissions to CREATE,ALTER.

Reply:
Dedicated user might be better.

Would suffixing '_super' to the pf.conf DB user sound fair to you ?

That way its kept simple

Reply:
That sounds good.

Reply:
What will happen if I create a survey table called node?

Reply:
All survey table names are always prefixed to prevent that issue

Reply:
OK cool

Reply:
Note left in b429fab for the rights as discussed